Class1.cs

```diff
@@ -1,22 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Rocket.API;
-using Rocket.Unturned;
-using Rocket.Core;
+﻿using Rocket.API;
 using UnityEngine;
-using Rocket.Unturned.Plugins;
-using SDG;
 using Rocket.Core.Plugins;
 using Rocket.Core.Logging;
 using Rocket.Unturned.Chat;
+using SDG.Unturned;
 
 //Yeha, we are on GitHub
 namespace Sadusko.DeathMessages
 {
-    public class PlayerDeath : RocketPlugin<DMC>
+    public class PlayerDeath : RocketPlugin<DMCconf>
```

Comment:
Actually why would I, this is not "improvements", this is basically changing.

DMC means DeathMessagesConfiguration.